api/src/main/resources/db/changelog/database/idC1_items.xml

```diff
@@ -0,0 +1,53 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.7"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.7
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.7.xsd">
+
+    <changeSet id="17" author="oscar">
+        <createTable tableName="items">
+            <column name="id" type="int" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="varchar(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="item_type" type="varchar(255)">
```

Comment:
ciekawostka: w bazach sql rozmiar bazy to ilosc rekordów * ich typowanie, a nie wsad.  Wiec optymalizacja polega na ustawieniu np. varchar(10) zamiast varchar(255) i smallint zamiast int

Reply:
jak próbuje zmienic np na smallint, lub thinyint, terminal wywala błąd że java i liquidbase nie znajdują się:

org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: wrong column type encountered in column [id] in table [items]; found [int2 (Types#SMALLINT)], but expecting [integer (Types#INTEGER)]

Reply:
to zrobię Pull-a na wszystkie zmiany, by zoptymalizować typy :)